tools/pyvmi/setup.py

```diff
@@ -1,21 +1,15 @@
-#!/usr/bin/env python
-import commands
-from distutils.core import setup, Extension
+#!/usr/bin/env python3
 
-pkgconfig_cflags = commands.getoutput ("pkg-config --cflags glib-2.0 libvmi")
-pkgconfig_include_flags = commands.getoutput ("pkg-config --cflags-only-I glib-2.0 libvmi")
-pkgconfig_include_dirs = [ t[2:] for t in pkgconfig_include_flags.split() ]
-pkgconfig_lflags = commands.getoutput ("pkg-config --libs-only-l glib-2.0 libvmi")
-pkgconfig_libs = [ t[2:] for t in pkgconfig_lflags.split() ]
-pkgconfig_biglflags = commands.getoutput ("pkg-config --libs-only-L glib-2.0 libvmi")
-pkgconfig_ldirs = [ t[2:] for t in pkgconfig_biglflags.split() ]
 
-pyvmimod = Extension('pyvmi', sources=['pyvmi.c'],
-                    include_dirs = pkgconfig_include_dirs,
-                    library_dirs = pkgconfig_ldirs,
-                    libraries = pkgconfig_libs,
-                    extra_compile_args=[pkgconfig_cflags])
+from setuptools import setup
 
-setup(name='PyVmi', version='1.1',
-      description = 'Python interface to LibVMI',
-      ext_modules = [pyvmimod])
+
+setup(
+    name='PyVMI',
+    version='2.0',
+    description='Python interface to LibVMI',
+    setup_requires=["cffi>=1.0.0", "pkgconfig"],
+    install_requires=["cffi>=1.0.0"],
```

Comment:
Missing `docopt` dependency.

Reply:
`docopt` is only needed for the examples (`process-list` and `module-list`), but not for the `libvmi.py` module itself.

Reply:
yes, i'd rather that it not be expressed as a dependency for the python layer. Regarding the examples, I think that it would be better to use more standard imports that are more likely to be available (already) on a user's system -- getopt itself may not be as featureful, but it is guaranteed to be present